R/textstat_lexdiv.R

```diff
@@ -91,26 +106,26 @@
 #' (result <- textstat_lexdiv(mydfm, c("CTTR", "TTR", "U")))
 #' cor(textstat_lexdiv(mydfm, "all")[,-1])
 #' 
-textstat_lexdiv <- function(x, measure = c("all", "TTR", "C", "R", "CTTR", "U", "S", "Maas"), 
+textstat_lexdiv <- function(x, measure = c("all", "TTR", "C", "R", "CTTR", "U", "S", "K", "Vm", "Maas"), 
```

Comment:
This line omitted 'D' as an accepted argument for `measure`

Reply:
Good 👁 @jiongweilua. Travis fails in the CI only because you did not also rebuild the documentation (.Rd) files. Best to have these options checked so you don't have to try to remember to do it manually.
<img width="693" alt="screenshot 2018-11-13 09 08 28" src="https://user-images.githubusercontent.com/2182246/48402577-df3d2c80-e723-11e8-8586-5272c63747ce.png">